<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInscription</class>
 <widget class="QDialog" name="DialogInscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>541</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inscription</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelPseudo">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>80</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pseudo : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valPseudo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMdp">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>113</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valMdp">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>113</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMdp2">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>146</y>
     <width>216</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmation du mot de passe : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valMdp2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>146</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>179</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valNom">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>179</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrenom">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>212</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pr√©nom :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valPrenom">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>212</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMail">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>245</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse mail :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valMail">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>245</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMail2">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>278</y>
     <width>214</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmation de l'adresse mail :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valMail2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>278</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateNaissance">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>311</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAdresse">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>344</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valAdresse">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>344</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodePostal">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>377</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valCodePostal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>377</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelVille">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>410</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valVille">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInscription">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Inscription</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonValider">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="valDateNaissance">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2015</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>1900</year>
     <month>12</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
